CHANGELOG.md

```diff
@@ -5,14 +5,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Changed
```

Comment:
Dit was echt een ongelukkige change:
* Merge conflicten in CHANGELOG zijn al vreselijk, maar op deze manier lokken we het ook nog eens uit
* Een CHANGELOG.md blame geeft voor het issue wat al was toegevoegd (en dus is verhuisd) een verkeerd beeld.

Ik heb net 3 commits gerebased en ik begreep al niet waarom die CHANGELOG zoveel problemen gaf..

Reply:
Ik kon het niet aanzien ;-) https://github.com/dso-toolkit/dso-toolkit/commit/f69915337b807da6660d0b9fe22f4fbb8209b46a#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14